tensorboard/plugins/scalar/vz_line_chart/vz-line-chart.ts

```diff
@@ -68,14 +68,13 @@ Polymer({
     smoothingWeight: {type: Number, value: 0.6},
 
     /**
-     * The way to display the X values. Allows:
-     * - "step" - Linear scale using the  "step" property of the datum.
-     * - "wall_time" - Temporal scale using the "wall_time" property of the
-     * datum.
-     * - "relative" - Temporal scale using the "relative" property of the
-     * datum if it is present or calculating from "wall_time" if it isn't.
+     * A function to create a ChartHelpers.XComponents and accepts no arguments.
```

Comment:
Something happened to the grammar in this sentence :-). Maybe, "``A function that accepts no arguments and returns a `ChartHelpers.XComponents`.``"?

You can also write `@type {function(): ChartHelpers.XComponents}` instead to convey exactly the same information _and_ get static analysis.